### PR TITLE
OC-12136:user can not overwrite a common event repeat/non repeat occu…

### PR DESCRIPTION
**Pull request by Taoli2018**
…rrence if data entered on first try was entered wrong through import

### CODE REVIEW

**Comment by svadla-oc** [comment]
jenkins retest this please

File: src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java [hunk 1]
```diff
@@ -939,7 +942,26 @@ private Object validateStudyEvent(StudyEventDataBean studyEventDataBean, StudySu
                     studyEventDataBean.setStudyEventRepeatKey(String.valueOf(studyEvent.getSampleOrdinal()));
                     return studyEvent;
                 } else {
-                    studyEventDataBean.setStudyEventRepeatKey(String.valueOf(eventOrdinal));
+                	// Repeat Key, if found in xml file, then use it
+                	if (studyEventDataBean.getStudyEventRepeatKey() != null && !studyEventDataBean.getStudyEventRepeatKey().equals("")) {   
+                		eventObject = validateEventRepeatKeyIntNumber(studyEventDataBean.getStudyEventRepeatKey());
+                        if (eventObject instanceof ErrorObj) return eventObject;                                      		              	
+                	}else {
+                		studyEventDataBean.setStudyEventRepeatKey(String.valueOf(eventOrdinal));
```
Comment (svadla-oc) [inline src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java]: why are you setting the repeat key in xml to the next event ordinal?

Comment (Taoli2018) [reply src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java]: if no repeat key, then set the repeat key in xml to the next event ordinal is current existing  logic

Comment (svadla-oc) [reply src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java]: Got it. thanks

**Comment by svadla-oc** [comment]
👍

**Pull request merged**